Hummingbird.docc/Tutorials/Todos/Resources/code/Postgres/todos-postgres-02.sh

```diff
@@ -8,4 +8,6 @@ postgres=# \c hummingbird
 You are now connected to database "hummingbird" as user "user".
 hummingbird=# create role todos createrole login password 'todos';
 CREATE ROLE
+hummingbird=# grant all on schema public to todos;
+GRANT
```

Comment:
I'm sorry what is this addition for? It shouldn't be necessary

Reply:
Hmm. I got a permission error on table creation without the "grant all" statement. Perhaps it's a misconfiguration on my end.

Reply:
I just set this on a vanilla install of psql, no extra configuration. 

Given you have v16 did you set it up differently?